private/tools/java/rules/jvm/external/HasherTest.java

```diff
@@ -56,13 +61,78 @@ public void sha256_multipleFiles() throws IOException, FileNotFoundException {
     File file3 = writeFile("test-3.file", "Hello World!");
     String[] files = new String[]{file1.getPath(), file2.getPath(), file3.getPath()};
 
-    String result = Hasher.hashFiles(files);
+    String result = Hasher.hashFiles(Stream.of(files));
     String[] lines = result.split("\n");
     assertThat(lines, arrayWithSize(3));
 
-    checkLine(lines[0], "7f83b1657ff1fc53b92dc18148a1d65dfc2d4b1fa3d677284addd200126d9069", "test-1.file");
-    checkLine(lines[1], "334d016f755cd6dc58c53a86e183882f8ec14f52fb05345887c8a5edd42c87b7", "test-2.file");
-    checkLine(lines[2], "7f83b1657ff1fc53b92dc18148a1d65dfc2d4b1fa3d677284addd200126d9069", "test-3.file");
+    checkLine(lines[0], "7f83b1657ff1fc53b92dc18148a1d65dfc2d4b1fa3d677284addd200126d9069",
```

Comment:
could you run google-java-format on this?

Reply:
Done.